eng/DisableSourceControlManagement.targets

```diff
@@ -1,9 +1,5 @@
 <Project>
 
-  <PropertyGroup>
-    <DisableSourceControlManagementTargetsImported>true</DisableSourceControlManagementTargetsImported>
```

Comment:
Is this related to:

> Remove a few stable properties (ie in DisableSourceControlManagement.targets)

What is a stable property? This is just an include guard AFAIK.

Reply:
I searched through the whole dotnet organization, this property is dead: https://github.com/search?q=org%3Adotnet+DisableSourceControlManagementTargetsImported&type=Code